src/main/java/net/praqma/tracey/broker/impl/filters/PayloadJSONFilter.java

```diff
@@ -15,6 +15,9 @@
 public class PayloadJSONFilter implements TraceyFilter {
 
     private String pattern = "";
+    private String key = "";
```

Comment:
I cannot use them final because they are bounding with Jenkins jelly form. There is one trick we can do, but I think it is better to leave it to the refactoring issue.